sdk/webpubsub/Azure.Messaging.WebPubSub.Client/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
-### Features Added
-
+## 1.0.0 (Unreleased)
```

Comment:
Specify the release date by replacing the `Unreleased`, the format is `yyyy-mm-dd`.

Reply:
Updated @ArthurMa1978